### ISSUE TITLE
Issue driven development

### ISSUE THREAD

**Issue by ghost** [fresearchgroup/drupal-logger#1]
Currently we are not following issue driven development... However, it is a preferred way... 
For every new feature, there should be a corresponding issue beforehand and commit implementing that new feature should address this issue.

**Comment by surajkothawade** [comment]
The link to the clear log messages button issue 
https://www.drupal.org/node/2275743 
Should we add  issue #2275743 to the commit?

**Comment by ghost** [comment]
@Suraj12378 no... what I meant was, in this repository, we should have an issue before implementing the new feature and the commit should refer to that issue (issue in this repository)... I was not talking about the upstream issue

so what you can do is, create a issue in this repo and amend your commit message to refer to that issue according to the good commit message guidelines mentioned here http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html

and for features you'll be implementing in future... create an issue first
